_data/languages.yml

```diff
@@ -1,30 +1,1065 @@
---- 
+---
+-
+  code: af
+  name: Afrikaans
+  family: ine
+  scripts: [ Latn ]
+  typology:
+    - SVO, VSO
+    - analytic
+  variants: [ za ]
+
+-
+  code: ar
+  name: Arabic
+  family: sem
+  scripts: [ Arab, Syrc ]
+  typology:
+    - VSO
+    - fusional
+  variants: [ dz, bh, eg, iq, jo, kw, lb, ly, ma, om, qa, sa, sy, tn, ae, ye, sa ]
+
+-
+  code: az
+  name: Azerbaijani
+  family: trk
+  scripts: [ Latn, Cyrl ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ az ]
+
+-
+  code: be
+  name: Belarusian
+  family: sla
+  scripts: [ Cyrl ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ by ]
+
+-
+  code: bg
+  name: Bulgarian
+  family: sla
+  scripts: [ Cyrl ]
+  typology:
+    - SVO
+    - analytic
+  variants: [ bg ]
+
+-
+  code: bn
+  name: Bengali
+  family: inc
+  scripts: Beng
+  typology:
+    - SOV
+    - inflected
+  variants: [ bd, in ]
+
+-
+  code: bs
+  name: Bosnian
+  family: sla
+  scripts: [ Cyrl, Latn ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ ba ]
+
+-
+  code: ca
+  name: Catalan
+  family: roa
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - fusional
+  variants: [ es ]
+
+-
+  code: ceb
+  name: Cebuano
+  family: phi
+  scripts: [ Latn ]
+  typology:
+    - VSO
+    - inflected
+  variants: [ ph ]
+
+-
+  code: co
+  name: Corsican
+  family: roa
+  scripts: [ Latn ]
+  typology:
+    - SOV
+  variants: [ fr ]
+
+-
+  code: cs
+  name: Czech
+  family: zlw
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - fusional, inflected, agglutinative
+  variants: [ cz ]
+
+-
+  code: cy
+  name: Welsh
+  family: cel
+  scripts: [ Latn ]
+  typology:
+    - VSO
+    - inflected
+  variants: [ gb ]
+
+-
+  code: da
+  name: Danish
+  family: gmq
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - analytic
+  variants: [ dk ]
+
+-
+  code: de
+  name: German
+  family: gem
+  scripts: [ Latn ]
+  typology:
+    - SVO, VSO
+    - fusional
+  variants: [ at, de, li, lu, ch ]
+
+-
+  code: el
+  name: Greek
+  family: grk
+  scripts: [ Grek ]
+  typology:
+    - SVO
+    - fusional
+  variants: [ cy, gr ]
+
+-
+  code: en
+  name: English
+  family: gem
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - analytic
+  variants: [ au, bz, ca, 029, in, ie, jm, my, nz, ph, sg, za, tt, gb, us, zw ]
+
+-
+  code: eo
+  name: Esperanto
+  family: constructed language
+  scripts: [ Latn ]
+  typology:
+    - free word-order
```

Comment:
I would just leave it empty/null or put all - SVO, OVS, SOV, VOS

On the other hand, I'm very skeptical.  e.g. Russian can be any, but in reality, one way is more natural, one way is more marked, in most situations, so that's why it only has one listed.

Reply:
I haven't found consensus determining a main word order.

[en.wikibooks.org/wiki/Esperanto:_A_Complete_and_Comprehensive_Grammar/Word_order](https://en.wikibooks.org/wiki/Esperanto:_A_Complete_and_Comprehensive_Grammar/Word_order)
[esperanto.50webs.com/EsrGrammar-5_03.html](http://esperanto.50webs.com/EsrGrammar-5_03.html)